doc/source/data/examples/batch_training.ipynb

```diff
@@ -0,0 +1,1228 @@
+       "      <td>207.324870</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>3</td>\n",
+       "      <td>LinearRegression()</td>\n",
+       "      <td>1412.273083</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>4</td>\n",
+       "      <td>LinearRegression()</td>\n",
+       "      <td>382.897671</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>5</td>\n",
+       "      <td>LinearRegression()</td>\n",
+       "      <td>2515.787663</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>...</th>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>252</th>\n",
+       "      <td>259</td>\n",
+       "      <td>LinearRegression()</td>\n",
+       "      <td>495.479018</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>253</th>\n",
+       "      <td>260</td>\n",
+       "      <td>LinearRegression()</td>\n",
+       "      <td>506.609215</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>254</th>\n",
+       "      <td>261</td>\n",
+       "      <td>LinearRegression()</td>\n",
+       "      <td>684.562030</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>255</th>\n",
+       "      <td>262</td>\n",
+       "      <td>LinearRegression()</td>\n",
+       "      <td>389.035014</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>256</th>\n",
+       "      <td>263</td>\n",
+       "      <td>LinearRegression()</td>\n",
+       "      <td>359.866881</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>257 rows × 3 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "     location_id               model        error\n",
+       "0              1  LinearRegression()  1186.550249\n",
+       "1              2  LinearRegression()   207.324870\n",
+       "2              3  LinearRegression()  1412.273083\n",
+       "3              4  LinearRegression()   382.897671\n",
+       "4              5  LinearRegression()  2515.787663\n",
+       "..           ...                 ...          ...\n",
+       "252          259  LinearRegression()   495.479018\n",
+       "253          260  LinearRegression()   506.609215\n",
+       "254          261  LinearRegression()   684.562030\n",
+       "255          262  LinearRegression()   389.035014\n",
+       "256          263  LinearRegression()   359.866881\n",
+       "\n",
+       "[257 rows x 3 columns]"
+      ]
+     },
+     "execution_count": 24,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# sort values by location id\n",
+    "results_df = results.to_pandas(limit=float(\"inf\"))\n",
```

Comment:
Nit: The `.to_pandas()` API has a default limit of 100k, which should be more than enough after the `.map_groups()` operation.
```suggestion
    "results_df = results.to_pandas()\n",
```